                                 endian.patch
                                 ------------

If you get Warnings on compilation about redefined XXXX_ENDIAN, then you
have broken Include-Files from libc-4.6.27. You can fix this with:

   # cd /usr/include
   # patch -p0 -s < /tmp/endian.patch

Bernd.Eckenfels@inka.de

--- endian.h.org       Sat Jul 24 04:31:39 1993
+++ endian.h    Mon Feb 19 20:39:34 1996
@@ -34,10 +34,18 @@
 #include <bytesex.h>

 #ifdef __USE_BSD
+#ifndef LITTLE_ENDIAN
 #define        LITTLE_ENDIAN   __LITTLE_ENDIAN
+#endif
+#ifndef BIG_ENDIAN
 #define        BIG_ENDIAN      __BIG_ENDIAN
+#endif
+#ifndef PDP_ENDIAN
 #define        PDP_ENDIAN      __PDP_ENDIAN
+#endif
+#ifndef BYTE_ORDER
 #define        BYTE_ORDER      __BYTE_ORDER
+endif
 #endif

 #endif /* endian.h */
